src/Game.cpp

```diff
@@ -218,6 +218,7 @@ void Game::doEvents()
 	while(SDL_PollEvent(&event))
 	{
 		mInputManager.updateKeyByEvent(event);
+		mInputManager.updateMouseMovement(event);
```

Comment:
This function is for the future version that will support mouse tracking, for now it is not used.